app/bundles/CampaignBundle/EventListener/LeadSubscriber.php

```diff
@@ -112,7 +112,7 @@ private function addTimelineEvents(LeadTimelineEvent $event, $eventTypeKey, $eve
         if (!$event->isEngagementCount()) {
             foreach ($logs['results'] as $log) {
                 $template = (!empty($eventSettings['action'][$log['type']]['timelineTemplate']))
-                    ? $eventSettings['action'][$log['type']]['timelineTemplate'] : 'MauticCampaignBundle:SubscribedEvents\Timeline:index.html.php';
+                    ? $eventSettings['action'][$log['type']]['timelineTemplate'] : 'MauticCampaignBundle:SubscribedEvents\Timeline:index.html.twig';
```

Comment:
`LeadBundle` needs to be updated in order for this to be rendered correctly on the Contact details page.